client/src/store/modules/user.js

```diff
@@ -48,7 +48,6 @@ export default {
           userInfo = response.data.filter(x => x.id === user.user.uid)
```

Comment:
It's related to previous commit, just a suggestion. Would be great to extend Node.js route and find user by Id on that level. I that case we would fetch particular user on backend level and wouldn't need to query all users and filter them. Example could be found here GeekScore-2/routes/api/games.js where we fetch particular game by id. Again could be another PR if you want to cover this issue